packages/eas-cli/src/utils/profiles.ts

```diff
@@ -8,19 +11,51 @@ type EasProfile<T extends ProfileType> = T extends 'build'
 export type ProfileData<T extends ProfileType> = {
   profile: EasProfile<T>;
   platform: Platform;
+  resourceClass: BuildResourceClass;
   profileName: string;
 };
 
+function getBuildResourceClassForPlatform({
```

Comment:
Profiles are an abstraction for entities from eas.json. This code does not belong here.